src/components/Inputs/index.tsx

```diff
@@ -1,8 +1,8 @@
 export { HTTPMethodSelect } from './HTTPMethodSelect'
 export { NumberInput } from './NumberInput'
 export { MillisecondInput } from './MillisecondInput'
-export { SelectInput } from './SelectInput'
-export { TextInput } from './TextInput'
-export { TextAreaInput } from './TextAreaInput'
+export { SelectInput } from './SelectInput/SelectInput'
+export { TextInput } from './TextInput/TextInput'
+export { TextAreaInput } from './TextAreaInput/TextAreaInput'
```

Comment:
Prefer importing from index : 
```suggestion
export { SelectInput } from './SelectInput'
export { TextInput } from './TextInput'
export { TextAreaInput } from './TextAreaInput'
```